Userland/Utilities/image.cpp

```diff
@@ -44,6 +47,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto frame = TRY(decoder->frame(0)).image;
     Optional<ReadonlyBytes> icc_data = TRY(decoder->icc_data());
 
+    RefPtr<Core::MappedFile> icc_file;
+    if (!assign_color_profile_path.is_empty()) {
```

Comment:
huh, continuing the previous comment, this feels a bit awkward. Would we ever both `--assign-color-profile` and `--strip-color-profile`? Feels a bit easier to grok if we handled them exclusively (it would be nice if ArgsParser had mutually exclusive flag support):

```c++
Optional<ReadonlyBytes> icc_data;
RefPtr<Core::MappedFile> icc_file;

if (!assign_color_profile_path.is_empty()) {
    icc_file = TRY(Core::MappedFile::map(assign_color_profile_path));
    icc_data = icc_file->bytes();
} else if (!strip_color_profile) {
    icc_data = TRY(decoder->icc_data());
}
```

Reply:
Got it, makes much more sense with `--convert-to-color-space` in mind